run/cifs_archive/archive-clips.sh

```diff
@@ -81,9 +81,15 @@ connectionmonitor $$ &
 # new file name pattern, firmware 2019.*
 moveclips "$CAM_MOUNT/TeslaCam/SavedClips"
 
+# Create trigger file for SavedClips
+if [ ! -z ${ARCHIVE_TRIGGER_SAVED+x} ] ; then log "Creating SavedClips Trigger File: $ARCHIVE_MOUNT/SavedClips/${ARCHIVE_TRIGGER_SAVED}"; touch "$ARCHIVE_MOUNT/SavedClips/${ARCHIVE_TRIGGER_SAVED}"; fi
+
 # v10 firmware adds a SentryClips folder
 moveclips "$CAM_MOUNT/TeslaCam/SentryClips"
 
+# Create trigger file for SentryClips
+if [ ! -z ${ARCHIVE_TRIGGER_SENTRY+x} ] ; then log "Creating SentryClips Trigger File: $ARCHIVE_MOUNT/SentryClips/${ARCHIVE_TRIGGER_SENTRY}"; touch "$ARCHIVE_MOUNT/SentryClips/${ARCHIVE_TRIGGER_SENTRY}"; fi
+
```

Comment:
use if/then style used in rest of scripts, and put each command on a separate line